### PR TITLE
docs: english quickstart proofreading

### PR DESCRIPTION
**Pull request by TanZiYen**

### CODE REVIEW

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: docs/en/quickstart/openmldb_quickstart.md [hunk 1]
```diff
@@ -51,21 +51,21 @@ The following screenshots show the correct execution of the above docker command
 
 ![image-20220111142406534](https://github.com/4paradigm/openmldb-docs-zh/blob/28159f05d2903a9f96e78d3205bd8e9a2d7cd5c3/quickstart/images/cli.png)
 
-### 2.2 Basic usage process
+### 2.2 Basic Usage Process
 
-The workflow of the standalone version of OpenMLDB generally includes several stages: database and table establishment, data preparation, offline feature extraction, SQL solution online, and online real-time feature extraction.
+The workflow of the standalone version of OpenMLDB generally includes several stages: Database and table establishment, data preparation, offline feature extraction, SQL solution online, and online real-time feature extraction.
```
Comment (lumianph) [inline docs/en/quickstart/openmldb_quickstart.md]: Database => database
no need to capitalize after the colon

File: docs/en/quickstart/openmldb_quickstart.md [hunk 2]
```diff
@@ -190,15 +190,15 @@ The following screenshot shows the screen after the cluster version of the OpenM
 
 ![image-20220111141358808](https://github.com/4paradigm/openmldb-docs-zh/blob/28159f05d2903a9f96e78d3205bd8e9a2d7cd5c3/quickstart/images/cli_cluster.png)
 
-### 3.3 Basic usage process
+### 3.3 Basic Usage Process
 
-The workflow of the cluster version of OpenMLDB generally includes several stages: database and table creation, offline data preparation, offline feature extraction, SQL solution online, online data preparation, and online real-time feature extraction.
+The workflow of the cluster version of OpenMLDB generally includes several stages: Database and table creation, offline data preparation, offline feature extraction, SQL solution online, online data preparation, and online real-time feature extraction.
```
Comment (lumianph) [inline docs/en/quickstart/openmldb_quickstart.md]: database

File: docs/en/quickstart/openmldb_quickstart.md [hunk 3]
```diff
@@ -323,9 +323,9 @@ After waiting for the task to complete, preview the online data:
 
 Up to this point, you have completed all the development and deployment work based on the cluster version of OpenMLDB CLI, and have returned to the operating system command line.
 
-#### 3.3.7 Real-time feature extraction
+#### 3.3.7 Real-Time Feature Extraction
 
-Notice: warning:: According to the default deployment configuration, the http port for apiserver deployment is 9080.
+Notice: Warning: According to the default deployment configuration, the http port for apiserver deployment is 9080.
```
Comment (lumianph) [inline docs/en/quickstart/openmldb_quickstart.md]: remove "Warning:"

**Review by lumianph — CHANGES_REQUESTED** [review]
@TanZiYen

**Review by lumianph — APPROVED** [review]

**Pull request merged**